vm/class_test.go

```diff
@@ -453,6 +462,101 @@ func TestRequireFail(t *testing.T) {
 	if err.Message != expected {
 		t.Fatalf("Error message should be '%s'. got: %s", expected, err.Message)
 	}
+	vm.checkCFP(t, 0, 1)
+}
+
+func TestGeneralIsNilMethod(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected bool
+	}{
+		{`123.is_nil`, false},
+		{`"Hello World".is_nil`, false},
+		{`(2..10).is_nil`, false},
+		{`{ a: 1, b: "2", c: ["Goby", 123] }.is_nil`, false},
+		{`[1, 2, 3, 4, 5].is_nil`, false},
+		{`true.is_nil`, false},
+		{`String.is_nil`, false},
+		{`nil.is_nil`, true},
+	}
+
+	for i, tt := range tests {
+		vm := initTestVM()
+		evaluated := vm.testEval(t, tt.input)
+		checkExpected(t, i, evaluated, tt.expected)
+		vm.checkCFP(t, i, 0)
+	}
+}
+
+func TestGeneralIsNilMethodFail(t *testing.T) {
+	testsFail := []struct {
+		input  string
+		errMsg string
+	}{
+		{`123.is_nil("Hello")`, "ArgumentError: Expect 0 argument. got: 1"},
+		{`123.is_nil("Hello", "World")`, "ArgumentError: Expect 0 argument. got: 2"},
```

Comment:
I think we don't need to test so many cases for this method, especially for error conditions.